tests/testthat/test-make_p_fn_make_q_fn.R

```diff
@@ -23,649 +23,621 @@
+
+  p_actual <- make_p_fn(ps, qs)(test_qs)
+  p_expected <- (1 / 3) * as.numeric(test_qs >= 1.0) + (2 / 3) * as.numeric(test_qs >= 2.0)
+  # plot(test_qs, p_actual); lines(test_qs, p_expected)
+
+  q_actual <- make_q_fn(ps, qs)(test_ps)
+  q_expected <- ifelse(test_ps <= 1 / 3, 1.0, 2.0)
+  # plot(test_ps, q_actual); lines(test_ps, q_expected)
+
+  testthat::expect_equal(p_actual, p_expected, tolerance = 1e-3)
+  testthat::expect_true(all(diff(p_actual[order(test_qs)]) >= 0))
+
+  testthat::expect_equal(q_actual, q_expected, tolerance = 1e-3)
+  testthat::expect_true(all(diff(q_actual[order(test_ps)]) >= 0))
+
+  testthat::expect_equal(c(rep(1 / 3, sum(test_ps <= 1 / 3)),
+                           rep(1, sum(test_ps > 1 / 3))),
+                         make_p_fn(ps, qs)(q_actual),
+                         tolerance = 1e-3)
+  # Commenting this test out -- fails due to floating point precision
+  # testthat::expect_equal(...,
+  #                        make_q_fn(ps, qs)(p_actual),
+  #                        tolerance = 1e-3)
 })
 
 
-test_that("make_p_fn, make_q_fn work, no continuous component; two point masses, non-zero, duplicated values first only", {
-    ps <- seq(from = 0.1, to = 0.4, by = 0.1)
-    qs <- c(rep(1.0, 3), rep(2.0, 1))
-
-    test_ps <- sort(c(
-        1 / 3, 1.0,
-        seq(from = 0.01, to = 0.99, by = 0.01)))
-    test_qs <- c(
-        1, 2,
-        qnorm(seq(from = 0.01, to = 0.99, by = 0.01), mean = 1, sd = 2))
-
-    p_actual <- make_p_fn(ps, qs)(test_qs)
-    p_expected <- (1 / 3) * as.numeric(test_qs >= 1.0) +
-                  (2 / 3) * as.numeric(test_qs >= 2.0)
-    # plot(test_qs, p_actual); lines(test_qs, p_expected)
-
-    q_actual <- make_q_fn(ps, qs)(test_ps)
-    q_expected <- ifelse(test_ps <= 1 / 3, 1.0, 2.0)
-    # plot(test_ps, q_actual); lines(test_ps, q_expected)
-
-    testthat::expect_equal(p_actual, p_expected, tolerance = 1e-3)
-    testthat::expect_true(all(diff(p_actual[order(test_qs)]) >= 0))
-
-    testthat::expect_equal(q_actual, q_expected, tolerance = 1e-3)
-    testthat::expect_true(all(diff(q_actual[order(test_ps)]) >= 0))
-
-    testthat::expect_equal(c(rep(1 / 3, sum(test_ps <= 1 / 3)),
-                             rep(1, sum(test_ps > 1 / 3))),
-                           make_p_fn(ps, qs)(q_actual),
-                           tolerance = 1e-3)
-    # Commenting this test out -- fails due to floating point precision
-    # testthat::expect_equal(...,
-    #                        make_q_fn(ps, qs)(p_actual),
-    #                        tolerance = 1e-3)
-})
+test_that("make_p_fn, make_q_fn work, no continuous component;
+          two point masses, non-zero, duplicated values first only", {
+            ps <- seq(from = 0.1, to = 0.4, by = 0.1)
+            qs <- c(rep(1.0, 3), rep(2.0, 1))
 
+            test_ps <- sort(c(1 / 3, 1.0, seq(from = 0.01, to = 0.99, by = 0.01)))
+            test_qs <- c(1, 2, qnorm(seq(from = 0.01, to = 0.99, by = 0.01), mean = 1, sd = 2))
 
```

Comment:
The indentation situation here is a little funky.  Let's indent to 2 spaces to be consistent with the rest of the stuff.

Reply:
continues below

Reply:
The inconsistent indentation here is a result of the first line of the test being 121 characters when the limit is 120 (the other tests generally make the 120 limit). This is the cause of many of the indentation issues you've requested changes for. Would you like me to try and shorten the test names with abbreviations to get around the inconsistent indentation between tests?

Reply:
It seems like we should be able to deal with this by inserting some new lines, right?  Here's a different example:

```
test_that("make_d_fn generates error, no continuous component; 
          two point masses, non-zero, duplicated values first only", {
            ps <- seq(from = 0.1, to = 0.4, by = 0.1)
            qs <- c(rep(1.0, 3), rep(2.0, 1))
            expect_error(make_d_fn(ps, qs))
          })
```

which i would think we could update to
```
test_that(
  "make_d_fn generates error, no continuous component; two point masses, non-zero, duplicated values first only",
  {
    ps <- seq(from = 0.1, to = 0.4, by = 0.1)
    qs <- c(rep(1.0, 3), rep(2.0, 1))
    expect_error(make_d_fn(ps, qs))
  }
)
```

Reply:
(i'm also up for shortening the test names as needed)

Reply:
Ok, I've gone ahead and implemented your suggested solution with inserting some new lines